tests/parsing/test_where_filter_parsing.py

```diff
@@ -135,3 +139,17 @@ def test_where_filter_intersection_from_partially_deserialized_list_of_strings()
     parsed_model = ModelWithWhereFilterIntersection.parse_obj(obj)
 
     assert parsed_model.where_filter == expected_parsed_output
+
+
+def test_time_dimension_date_part() -> None:  # noqa
```

Comment:
Should we add tests without all the params?

Reply:
➕